modules/ROOT/pages/functions/dev_guide/nodejs/develop-nodejs.adoc

```diff
@@ -0,0 +1,7 @@
+[id="develop-nodejs"]
+= Developing Node.js functions
+
+toc::[]
+
+include::nodejs-template.adoc[leveloffset=+1]
+include::../../../functions/user_guide/invoking-functions.adoc[leveloffset=+1]
```

Comment:
Need to figure out what content is generic wrt invoking functions vs what are specific runtime examples